src/toil/fileStore.py

```diff
@@ -47,11 +51,13 @@
     'kwargs'))
 
 
-class FileStore(object):
-    '''
-    Class used to manage temporary files, read and write files from the job store\
-    and log messages, passed as argument to the :func:`toil.job.Job.run` method.
-    '''
+class AbstractFileStore(object):
+    """
+    An abstract (as far as Python currently allows) base class to represent the interface between a
```

Comment:
I actually copied that first line from abstractBatchSystem (which I used as a template for an abstract class)